compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6495,6 +6495,112 @@ OMR::Z::TreeEvaluator::bstoreEvaluator(TR::Node * node, TR::CodeGenerator * cg)
    return NULL;
    }
 
+TR::Register *
+OMR::Z::TreeEvaluator::irdbarEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   if (node->getSymbolReference()->getSymbol()->isStatic())
+      cg->decReferenceCount(node->getFirstChild());
+   return TR::TreeEvaluator::iloadEvaluator(node,cg);
+   }
```

Comment:
It appears that now all codegens (bar ARM) implement these evaluators in the exact same way. Would it be possible to move these evaluators up to the common code generator and we all use the same common implementation?

Reply:
Also space needed between the two arguments `node,cg`

Reply:
This should no longer be an issue in the latest version.